### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by QuestYouCraft**

### CODE REVIEW

**Comment by QuestYouCraft** [comment]
I'm update README.md again;
- added content map

**Comment by a1ive** [comment]
深度截图_选择区域_20200408101953
Items are not aligned?

File: README.md [hunk 1]
```diff
@@ -1,11 +1,27 @@
-[简体中文](https://a1ive.github.io/grub2-filemanager/) 
+[Website of repo](https://a1ive.github.io/grub2-filemanager/)
```
Comment (a1ive) [inline README.md:1]: This website is for Chinese users.

File: README.md [hunk 2]
```diff
@@ -55,6 +71,8 @@ GRUB2: https://github.com/a1ive/grub
 [Crowdin](https://crowdin.com/project/grub2-filemanager) 
 
 ## Similar projects 
+
+* [GrubFM Installer](https://github.com/QuestYouCraft/grubfm-installer)
```
Comment (a1ive) [inline README.md:75]: Use tab instead of space?

File: README.md (hunk 1 above)
Comment (QuestYouCraft) [reply README.md:1]: > 
> 
> This website is for Chinese users.

OK

File: README.md (hunk 2 above)
Comment (QuestYouCraft) [reply README.md:75]: Yes

Comment (QuestYouCraft) [reply README.md:75]: > 
> 
> Use tab instead of space?

Yes

**Comment by QuestYouCraft** [comment]
> 
> 
> 深度截图_选择区域_20200408101953
> Items are not aligned?
Fixed

**Pull request merged**